doc/user/content/sql/create-source/avro-kafka.md

```diff
@@ -107,13 +107,13 @@ This creates a source that...
   `localhost:9092`.
 - Is append-only.
 
-### Connecting to a Kafka broker using SASL PLAIN authentication
+### Connecting to a Kafka broker using SASL authentication
```

Comment:
There was a link to this in the release notes––would you mind updating that, as well? You can see any broken links by going to the bottom of the PR and clicking, next to **buildkite/tests/briefcase-lint-docs**, **Details**.